style.css

```diff
@@ -0,0 +1,416 @@
+	letter-spacing: 0.1rem;
+}
+
+.menu-item {
+	font-weight: 500;
+}
+
+.menu-item:hover {
+	border-bottom: 1px solid #09003d;
+}
+
+.logo {
+	font-size: 1.3rem;
+	font-weight: 700;
+}
+
+.logo:hover {
+	color: #ff0062;
+}
+
+/*FIRST SECTION*/
+.intro {
+	background-image: url('https://images.unsplash.com/photo-1542831371-29b0f74f9713?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1350&q=80');
+	background-repeat: no-repeat;
+	background-size: cover;
+	}
+
+.intro-content {
+	color: white;
+	background: rgba(9, 0, 61,0.8);
+	height: 100%;
+	padding: 150px 100px;
+}
+
+.intro-wrapper {
+	width: 60%;
+}
+
+h1 {
+	font-size: 3.5rem;
+	font-weight: 700;
+}
+
+h1 span {
+	color: #ff0062;
+	white-space: nowrap;
+}
+
+.subtitle {
+	font-weight: 700;
+	font-size: 3rem;
+	color: #FED017;
+	margin: 60px 0 40px 0
+}
+
+.desc-intro{
+	line-height: 2.4rem;
+	font-size: 1.7rem;
+	font-weight: 400;
+	margin-bottom: 100px;
+}
+
+.cta-button a {
+	font-size: 1.2rem;
+	font-weight: 600;
+	padding: 15px 25px;
+	background: #61FED2;
+	border-radius: 40px;
+	color: #09003d;
+}
+
+.coding-lang-wrapper {
+	background:  #F1F4FF;
+	padding: 50px 0;
+	display: flex;
+	justify-content: space-evenly;
+	align-items: center;
+}
+
+.coding-lang-wrapper i {
+	font-size: 8rem;
+}
+
+/*     */
+
+h2 {
+	font-size: 3rem;
+	font-weight: 600;
+	line-height: 3rem;
+	scroll-margin: 20px;
+}
+
+.desc-about-me-wrapper p, .desc-portfolio-wrapper p, .contact p {
+	font-size: 1.4rem;
+	line-height: 2.4rem;
+	font-weight: 300;
+}
+
+.desc-about-me-wrapper p, .desc-portfolio-wrapper p, .desc-about-me-wrapper h2, .desc-portfolio-wrapper h2, .contact-wrapper {
+	width: 70%;
+	margin-right: 0;
+}
+
+.project-details {
+	position: absolute;
+	bottom: 0;
+	display:flex;
+	justify-content: space-between;
+	padding: 10px;	
+	width: 100%;
+}
+
+.project-details i {
+	font-size: 2rem;
+	color: white;
+}
+
+h3.project-title {
+	font-size: 2rem;
+	font-weight: 500;
+	color: white;	
+	margin: 0;
+	display: inline;
+}
+
+.desc-portfolio-wrapper {
+	margin-bottom: 20px;
+}
+ 
+ .portfolio {
+	color: black;
+	background: #F1F4FF;
+	padding-bottom: 0;
+}
+
+.projects-wrapper {
+	display:flex;
+	justify-content: space-between;
+	flex-wrap: wrap;
+	margin-top: 100px;
+	padding: 0;
+}
+
+.project-display {
+	position: relative;
+	width: 50%;
+	height: 200px;
+	border: 5px solid #F1F4FF;
+}
+
+.project-screenshot {
+	object-fit: cover;
+  	object-position: 50% 20%;
+	position: absolute;
+	max-width: 100%;
+	width: 100%;
+	height:100%;
+}
+
+.contact {
+	display: flex;
+	flex-direction: column;
+	justify-content: flex-start;
+	align-items: flex-start;
+	background: #09003d;
+	color: #ffffff;
+}
+
+.contact-icons-wrapper {
+	margin: 20px 0;
+}
+
+.icon {
+	color: inherit;
+	margin: 0 5px 0 5px;
+	font-size: 3rem;
+}
+
+
+/*CONTACT FORM  IN FOOTER
+- Added an hidden iframe to avoid opening a new window after submiting
+
+*/
+.hide { 
+	display:none; 
+}
```

Comment:
this will be deleted anyway but that's not a very semantic class name because it describes exactly what the css does - i would maybe call it `secret-iframe` or something like that

Reply:
Right, thanks!